.github/workflows/coverage.yml

```diff
@@ -12,6 +12,16 @@ jobs:
         uses: actions/setup-node@v2-beta
         with:
           node-version: "*"
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
```

Comment:
Does this also work with Yarn 2?

Reply:
It works on Yarn 2 only: https://github.com/actions/cache/blob/main/examples.md#node---yarn-2